core/src/components/segment/segment.tsx

```diff
@@ -49,6 +49,16 @@ export class Segment implements ComponentInterface {
    */
   @Prop() scrollable = false;
 
+  /**
+   * If `false`, the gesture to drag the indicator between the buttons will be disabled.
```

Comment:
Can we re-word this so that it explain it more "positively"? Something like "If `true`, users will be able to swipe between segment buttons to activate them".

Reply:
I will change it to yours!